src/components/AlertMessage.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from "react-native";
+
+const styles = StyleSheet.create({
+  outer:{
+    flex:1,
+    backgroundColor: '#b20700',
+  },
+  titleContainer:{
+    flex: 0.5,
+    justifyContent: 'center',
+  },
+  textContainer: {
+    flex: 0.5,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  buttonContainer:{
+    flex:1,
+    justifyContent:'center',
+    alignItems: 'center',
+  },
+  welcome: {
+    marginLeft: '10%',
+    textAlign: 'left',
+    fontSize: 50,
+    color: '#FFFFFF',
+  },
+  paragraph: {
+    marginVertical: "10%",
+    marginHorizontal: "10%",
+    fontSize: 18,
+    alignItems: 'center',
+    textAlign: 'left',
+    color: '#FFFFFF',
+  },
+  button: {
+    position: 'absolute',
+    width: "80%",
+    height:"3em",
+    borderRadius:10,
+    justifyContent:'center',
+    backgroundColor: '#FFFFFF',
+    bottom:25,
+  },
+  buttonText:{
+    textAlign:'center',
+
+  },
+});
+
+const AlertMessage = props => {
+  const {message} = props;
+
+  if ({message}) {
```

Comment:
Unnecessary {} wrapping this message, this is constructing an object {"message": message} and will always be true. `!!{}` demonstrates how an empty object evaluates to true.